sql/drop.go

```diff
@@ -0,0 +1,48 @@
+package sql
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/keys"
+	"github.com/cockroachdb/cockroach/sql/parser"
+	"github.com/cockroachdb/cockroach/structured"
+)
+
+// DropTable drops a table.
+func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
```

Comment:
@MBerhault  After the table is dropped, the test cannot verify the data is removed from the sql interface. So maybe I should add another test in drop_test to check the data is removed?